components/index.html

```diff
@@ -1121,7 +1121,7 @@ <h4 class="footer__headline">Support</h4>
             <div class="col-md-6">
                 <h4 class="footer__headline">Copyright</h4>
                 <p>
-                    &copy; 2022 by <a href="https://getlaminas.org/">Laminas Project</a> a Series of LF Projects, LLC.
+                    &copy; 2023 by <a href="https://getlaminas.org/">Laminas Project</a> a Series of LF Projects, LLC.
```

Comment:
Can we just drop the year?

Reply:
If we follow the requirements for the copyright notice (US), we do not need to update the year at all, as this information should include "the year of first publication".

Reply:
can use javascript? 
wrote once and that's it

Reply:
so we just roll with the date where laminas was first created?

Reply:
The problem is that everyone thinks the website is outdated. This can be observed on other websites and I think this is also the reason why @Arhell reported this as a problem.
Personally, I often read it as outdated too, but how much do I have to do with US copyright law?! 😉

Reply:
> can use javascript?

Can be automated via [another build script](https://github.com/laminas/laminas.github.io/tree/master/build) or [GitHub actions](https://github.com/laminas/laminas.github.io/projects/1).

Reply:
If we follow companies like Adobe, the current year seems to be right:

> Copyright © 2023 Adobe. All rights reserved.

https://www.adobe.com